Unable to import archive due to validation error on ActorCatalog

## Environment
- **Airbyte version**: 0.35.49-alpha
- **OS Version / Instance**: macOS
- **Deployment**: Docker
- **Severity**: Medium
- **Step where error happened**: Import configuration archive 

## Current Behavior
A JSON validation exception is thrown when importing an archive recently produced by an export. The exceptions seems to be related to https://github.com/airbytehq/airbyte/pull/10326/files

If i comment out the code from https://github.com/airbytehq/airbyte/blob/d19754c169f5188c3926492d1ddf7b595c6d1624/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java#L1558 all the way to Line 1572, then the import works.

## Expected Behavior
Successful import

## Logs
*If applicable, please upload the logs from the failing operation. 
For sync jobs, you can download the full logs from the UI by going to the sync attempt page and 
clicking the download logs button at the top right of the logs display window.*

<details>
<summary>LOG</summary> 

```
airbyte-server      | 2022-03-10 16:05:32 ERROR i.a.s.h.ArchiveHandler(importInternal):136 - Import failed
airbyte-server      | java.lang.RuntimeException: io.airbyte.validation.json.JsonValidationException: json schema validation failed when comparing the data to the json schema. 
airbyte-server      | Errors: $.id: is missing but it is required, $.catalog: is missing but it is required, $.catalogHash: is missing but it is required 
airbyte-server      | Schema: 
airbyte-server      | {
airbyte-server      |   "$schema" : "http://json-schema.org/draft-07/schema#",
airbyte-server      |   "$id" : "https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/AttemptFailureSummary.yaml",
airbyte-server      |   "title" : "ActorCatalog",
airbyte-server      |   "description" : "Catalog of an actor.",
airbyte-server      |   "type" : "object",
airbyte-server      |   "additionalProperties" : false,
airbyte-server      |   "required" : [ "id", "catalog", "catalogHash" ],
airbyte-server      |   "properties" : {
airbyte-server      |     "id" : {
airbyte-server      |       "type" : "string",
airbyte-server      |       "format" : "uuid"
airbyte-server      |     },
airbyte-server      |     "catalog" : {
airbyte-server      |       "type" : "object",
airbyte-server      |       "existingJavaType" : "com.fasterxml.jackson.databind.JsonNode"
airbyte-server      |     },
airbyte-server      |     "catalogHash" : {
airbyte-server      |       "type" : "string"
airbyte-server      |     }
airbyte-server      |   }
airbyte-server      | }
airbyte-server      |   at io.airbyte.server.ConfigDumpImporter.lambda$readConfigsFromArchive$2(ConfigDumpImporter.java:203) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197) ~[?:?]
airbyte-server      |   at java.util.ArrayList$Itr.forEachRemaining(ArrayList.java:1003) ~[?:?]
airbyte-server      |   at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845) ~[?:?]
airbyte-server      |   at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
airbyte-server      |   at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
airbyte-server      |   at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?]
airbyte-server      |   at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?]
airbyte-server      |   at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
airbyte-server      |   at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
airbyte-server      |   at io.airbyte.config.persistence.DatabaseConfigPersistence.lambda$replaceAllConfigs$89(DatabaseConfigPersistence.java:1560) ~[io.airbyte.airbyte-config-persistence-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.db.Database.lambda$transaction$0(Database.java:31) ~[io.airbyte.airbyte-db-lib-0.35.49-alpha.jar:?]
airbyte-server      |   at org.jooq.impl.DefaultDSLContext.lambda$transactionResult0$0(DefaultDSLContext.java:564) ~[jooq-3.13.4.jar:?]
airbyte-server      |   at org.jooq.impl.Tools$12$1.block(Tools.java:4926) ~[jooq-3.13.4.jar:?]
airbyte-server      |   at java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3463) ~[?:?]
airbyte-server      |   at java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3434) ~[?:?]
airbyte-server      |   at org.jooq.impl.Tools$12.get(Tools.java:4923) ~[jooq-3.13.4.jar:?]
airbyte-server      |   at org.jooq.impl.DefaultDSLContext.transactionResult0(DefaultDSLContext.java:616) ~[jooq-3.13.4.jar:?]
airbyte-server      |   at org.jooq.impl.DefaultDSLContext.transactionResult(DefaultDSLContext.java:533) ~[jooq-3.13.4.jar:?]
airbyte-server      |   at io.airbyte.db.Database.transaction(Database.java:31) ~[io.airbyte.airbyte-db-lib-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.db.ExceptionWrappingDatabase.transaction(ExceptionWrappingDatabase.java:31) ~[io.airbyte.airbyte-db-lib-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.config.persistence.DatabaseConfigPersistence.replaceAllConfigs(DatabaseConfigPersistence.java:1468) ~[io.airbyte.airbyte-config-persistence-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.config.persistence.ValidatingConfigPersistence.replaceAllConfigs(ValidatingConfigPersistence.java:100) ~[io.airbyte.airbyte-config-persistence-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.config.persistence.ConfigRepository.replaceAllConfigs(ConfigRepository.java:873) ~[io.airbyte.airbyte-config-persistence-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.ConfigDumpImporter.importConfigsFromArchive(ConfigDumpImporter.java:185) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.ConfigDumpImporter.importDataWithSeed(ConfigDumpImporter.java:130) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.handlers.ArchiveHandler.lambda$importData$0(ArchiveHandler.java:102) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.handlers.ArchiveHandler.importInternal(ArchiveHandler.java:133) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.handlers.ArchiveHandler.importData(ArchiveHandler.java:102) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.apis.ConfigurationApi.lambda$importArchive$75(ConfigurationApi.java:711) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.apis.ConfigurationApi.execute(ConfigurationApi.java:735) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.apis.ConfigurationApi.importArchive(ConfigurationApi.java:711) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
airbyte-server      |   at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
airbyte-server      |   at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
airbyte-server      |   at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
airbyte-server      |   at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:124) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:167) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:79) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:469) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:391) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:80) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:253) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248) ~[jersey-common-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244) ~[jersey-common-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.internal.Errors.process(Errors.java:292) ~[jersey-common-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.internal.Errors.process(Errors.java:274) ~[jersey-common-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.internal.Errors.process(Errors.java:244) ~[jersey-common-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265) ~[jersey-common-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:232) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:680) ~[jersey-server-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394) ~[jersey-container-servlet-core-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346) ~[jersey-container-servlet-core-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:366) ~[jersey-container-servlet-core-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:319) ~[jersey-container-servlet-core-2.31.jar:?]
airbyte-server      |   at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205) ~[jersey-container-servlet-core-2.31.jar:?]
airbyte-server      |   at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:763) ~[jetty-servlet-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:569) ~[jetty-servlet-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:233) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1377) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:188) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:507) ~[jetty-servlet-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:186) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1292) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:127) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.Server.handle(Server.java:501) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.HttpChannel.lambda$handle$1(HttpChannel.java:383) ~[jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:556) [jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:375) [jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:273) [jetty-server-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:311) [jetty-io-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:105) [jetty-io-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.io.ChannelEndPoint$1.run(ChannelEndPoint.java:104) [jetty-io-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:336) [jetty-util-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:313) [jetty-util-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:171) [jetty-util-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:129) [jetty-util-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:375) [jetty-util-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:806) [jetty-util-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:938) [jetty-util-9.4.31.v20200723.jar:9.4.31.v20200723]
airbyte-server      |   at java.lang.Thread.run(Thread.java:833) [?:?]
airbyte-server      | Caused by: io.airbyte.validation.json.JsonValidationException: json schema validation failed when comparing the data to the json schema. 
airbyte-server      | Errors: $.id: is missing but it is required, $.catalog: is missing but it is required, $.catalogHash: is missing but it is required 
airbyte-server      | Schema: 
airbyte-server      | {
airbyte-server      |   "$schema" : "http://json-schema.org/draft-07/schema#",
airbyte-server      |   "$id" : "https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/AttemptFailureSummary.yaml",
airbyte-server      |   "title" : "ActorCatalog",
airbyte-server      |   "description" : "Catalog of an actor.",
airbyte-server      |   "type" : "object",
airbyte-server      |   "additionalProperties" : false,
airbyte-server      |   "required" : [ "id", "catalog", "catalogHash" ],
airbyte-server      |   "properties" : {
airbyte-server      |     "id" : {
airbyte-server      |       "type" : "string",
airbyte-server      |       "format" : "uuid"
airbyte-server      |     },
airbyte-server      |     "catalog" : {
airbyte-server      |       "type" : "object",
airbyte-server      |       "existingJavaType" : "com.fasterxml.jackson.databind.JsonNode"
airbyte-server      |     },
airbyte-server      |     "catalogHash" : {
airbyte-server      |       "type" : "string"
airbyte-server      |     }
airbyte-server      |   }
airbyte-server      | }
airbyte-server      |   at io.airbyte.validation.json.JsonSchemaValidator.ensure(JsonSchemaValidator.java:68) ~[io.airbyte-airbyte-json-validation-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.ConfigDumpImporter.validateJson(ConfigDumpImporter.java:215) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   at io.airbyte.server.ConfigDumpImporter.lambda$readConfigsFromArchive$2(ConfigDumpImporter.java:200) ~[io.airbyte-airbyte-server-0.35.49-alpha.jar:?]
airbyte-server      |   ... 83 more

```

</details>

## Steps to Reproduce
1.
2.
3.

## Are you willing to submit a PR?
<!--- 
We accept contributions! 
Don't feel pressured, but if you want to contribute we can help you by giving some tips, 
highlighting the necessary code change or explaining any relevant point your feature will impact.
You can also send questions on #dev Slack channel.

We understand if you can't submit a PR and we're tremendously grateful
that you've already contributed by suggesting a new feature.
-->
Remove this with your answer.

--- a/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java
+++ b/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java
@@ -1668,7 +1668,7 @@ public class DatabaseConfigPersistence implements ConfigPersistence {
               .map(Jsons::jsonNode));
     }
     final List<ConfigWithMetadata<ActorCatalog>> actorCatalogWithMetadata = listActorCatalogWithMetadata();
-    if (!standardSyncStateWithMetadata.isEmpty()) {
+    if (!actorCatalogWithMetadata.isEmpty()) {
       result.put(ConfigSchema.ACTOR_CATALOG.name(),
           actorCatalogWithMetadata
               .stream()
@@ -1676,7 +1676,7 @@ public class DatabaseConfigPersistence implements ConfigPersistence {
               .map(Jsons::jsonNode));
     }
     final List<ConfigWithMetadata<ActorCatalogFetchEvent>> actorCatalogFetchEventWithMetadata = listActorCatalogFetchEventWithMetadata();
-    if (!standardSyncStateWithMetadata.isEmpty()) {
+    if (!actorCatalogFetchEventWithMetadata.isEmpty()) {
       result.put(ConfigSchema.ACTOR_CATALOG_FETCH_EVENT.name(),
           actorCatalogFetchEventWithMetadata
               .stream()

--- a/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java
+++ b/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java
@@ -1670,7 +1670,7 @@ public class DatabaseConfigPersistence implements ConfigPersistence {
     final List<ConfigWithMetadata<ActorCatalog>> actorCatalogWithMetadata = listActorCatalogWithMetadata();
     if (!standardSyncStateWithMetadata.isEmpty()) {
       result.put(ConfigSchema.ACTOR_CATALOG.name(),
-          standardSyncStateWithMetadata
+          actorCatalogWithMetadata
               .stream()
               .map(ConfigWithMetadata::getConfig)
               .map(Jsons::jsonNode));
@@ -1678,7 +1678,7 @@ public class DatabaseConfigPersistence implements ConfigPersistence {
     final List<ConfigWithMetadata<ActorCatalogFetchEvent>> actorCatalogFetchEventWithMetadata = listActorCatalogFetchEventWithMetadata();
     if (!standardSyncStateWithMetadata.isEmpty()) {
       result.put(ConfigSchema.ACTOR_CATALOG_FETCH_EVENT.name(),
-          standardSyncStateWithMetadata
+          actorCatalogFetchEventWithMetadata
               .stream()
               .map(ConfigWithMetadata::getConfig)
               .map(Jsons::jsonNode));

